decidim-core/app/views/decidim/endorsements/_identity.html.erb

```diff
@@ -1,9 +1,4 @@
-<li class="<%= selected ? "selected" : "" %>" data-method="<%= http_method.to_s.upcase %>" data-create_url="<%= create_url %>" data-destroy_url="<%= destroy_url %>">
-  <div class="author-data author-data--small">
-    <div class="author-data__main">
-      <%== cell("decidim/author", identity, skip_profile_link: true).profile %>
-    </div>
-  </div>
-
-  <%= icon("circle-check", class: "icon--big #{selected ? '' : 'invisible'}") %>
-</li>
+<button type="button" class="<%= selected ? "is-selected" : "" %>" data-user-identity data-method="<%= http_method.to_s.upcase %>" data-create_url="<%= create_url %>" data-destroy_url="<%= destroy_url %>">
+  <%== cell("decidim/author", identity, skip_profile_link: true).profile %>
+  <%= icon("checkbox-circle-line") %>
```

Comment:
I would suggest not showing the "check" icon with the hover/active style.

This is what the modal looks like after I've clicked the second item (group) to dislike as group:

![Dislike icon](https://user-images.githubusercontent.com/864340/206746741-c7436310-3f90-44c0-9009-a75ecafcfae2.png)

Note that I am not hovering that item in the screenshot. This is because the item stays as active after disliking (which is good) but it is confusing that it still shows the check mark even after I just removed the group like.